docs/pages/includes/tctl.mdx

```diff
@@ -1,35 +1,28 @@
-Make sure you can connect to Teleport. Log in to your cluster using `tsh`, then use `tctl`
-remotely:
+To check that you can connect to your Teleport cluster, sign in with `tsh`, then 
+verify that you can run `tctl` commands on the host using your current credentials:
 
 {/* Ignoring scope linting since we use this partial throughout the docs and
 cannot guarantee that it will line up with a page's configured scopes*/}
 {/*lint ignore scopes*/} 
-<ScopedBlock scope={["oss", "enterprise"]}>
 
 ```code
-$ tsh login --proxy=teleport.example.com --user=email@example.com
+$ tsh login --proxy=<Var name="teleport.example.com" /> --user=<Var name="email@example.com" />
 $ tctl status
-# Cluster  teleport.example.com
+# Cluster  (=teleport.url=)
 # Version  (=teleport.version=)
 # CA pin   (=presets.ca_pin=)
 ```
 
-You can run subsequent `tctl` commands in this guide on your local machine.
-
-For full privileges, you can also run `tctl` commands on your Auth Service host.
+<ScopedBlock scope={["oss", "enterprise"]}>
+If you can connect to the cluster and run the `tctl` command,
+you can run subsequent `tctl` commands on the host computer.
 
+For full privileges, you can also run `tctl` commands on your 
+Teleport Auth Service host.
 </ScopedBlock>
+
 {/*lint ignore scopes*/}
 <ScopedBlock scope={["cloud","team"]}>
-
-```code
-$ tsh login --proxy=myinstance.teleport.sh --user=email@example.com
-$ tctl status
-# Cluster  myinstance.teleport.sh
-# Version  (=cloud.version=)
-# CA pin   sha256:sha-hash-here
-```
-
-You must run subsequent `tctl` commands in this guide on your local machine.
-
+After you connect to your Teleport cluster instance, you must run subsequent `tctl` commands 
+on the host computer.
 </ScopedBlock>
```

Comment:
I'm trying to phase out `ScopedBlock`s since users tend to get confused by the scope switcher. What would you think of one sentence in all scopes saying something like,

"If you host your own Teleport cluster, you can run `tctl` on the same host as the Auth Service for full permissions."